swan/templates/cvmfs/automount-pvc.yaml

```diff
@@ -1,9 +1,3 @@
-apiVersion: storage.k8s.io/v1
```

Comment:
Did you try removing the storage class that was installed, and see if the cvmfs chart itself adds it?

Reply:
Yes, it adds it to the cluster from what I saw